app/views/user_transactions/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<section class="main-section">
+<h1 class="header" >NEW TRANSACTION</h1>
+<%= render "form", user_transaction: @user_transaction %>
+
+<div class="links-wrapper"> <%= link_to "Back to Transactions", group_user_transactions_path, class:'link' %></div>
+
+</section>
```

Comment:
- [ ] Kindly also improve the styling of the new transaction page based on the feedback here: